src/exporter.js

```diff
@@ -133,9 +136,22 @@ export default class {
   _tagsToStr(tags=[]){
     return tags.map(tag => tag.replace(/ /g, '_')).join(' ');
   }
-}
 
-export const rand = () => Math.random() * 100000000 | 0;
+  _getId(table, col, ts) {
+    const query = `SELECT ${col} from ${table} WHERE ${col} >= ${ts}`;
```

Comment:
If I'll use `LIMIT 1` -  I need to make multiple queries ( to be sure that `ts+1` not presented in column already ). That's the reason why I select without limit - make only single query and increment/check inside results

About prepared statements - ok. Thx